Dockerfile

```diff
@@ -1,8 +1,15 @@
-FROM python:3.10
+FROM python:3.10.6
 
 ENV POETRY_VERSION=1.0.3
 ENV DCM2NIIX_VERSION=v1.0.20190902
 
+# Install Node.js and npm
+RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - \
+    && apt-get install -y nodejs
+
+# Install bids-validator using npm
+RUN npm install -g bids-validator
```

Comment:
We are installing bids-validator in the action. We don't need it baked into the container.

We'll only need to do this if the action doesn't work